src/test/java/org/jenkinsci/plugins/gitclient/MergeCommandTest.java

```diff
@@ -225,6 +239,12 @@ public void testRecursiveStrategy() throws GitException, InterruptedException {
         assertTrue("README 1 missing on master branch", readmeOne.exists());
     }
 
+    @Test
+    public void testRecursiveTheirsStrategy() throws GitException, InterruptedException {
+        mergeCmd.setStrategy(MergeCommand.Strategy.RECURSIVE_THEIRS).setRevisionToMerge(commit1Branch2).execute();
+        assertTrue("branch 2 commit 1 not on master branch after merge", git.revList("master").contains(commit1Branch2));
```

Comment:
Thanks for writing the test to confirm that the strategy option is accepted.

Would you be willing to include an assertion in the test which confirms that the behavior of the THEIRS option was as expected?  For example, could you assert that the contents of the merged file were as expected?